include/OpenSpaceToolkit/Physics/Coordinate/Spherical/LLA.hpp

```diff
@@ -67,6 +105,13 @@ class LLA
         const LLA& aLLA, const Length& anEllipsoidEquatorialRadius, const Real& anEllipsoidFlattening
     ) const;
 
+    /// @brief                  Calculate the distance between this LLA coordinate and another LLA coordinate
+    ///
+    /// @param                  [in] aLLA A LLA coordinate
+    /// @return
+
+    Length calculateDistanceTo(const LLA& aLLA) const;
```

Comment:
I think we should stick to a rule that we always expose the explicit variant too? (if we use default args, then my point is moot)